.gitignore

```diff
@@ -9,6 +9,11 @@
 ### Sphinx/Docs ###
 /docs/_build
 
```

Comment:
Running the Makefile creates these directories in the repository, you probably don't want to commit them ever, so I thought that adding these here made sense.